src/main/resources/mybatis/mappers/navigate.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd"> 
+
+<mapper namespace="navigate">
+
+	<select id="navigate" resultType="java.util.LinkedHashMap" >
+		select nhdplus_navigation.navigate(                
+             pnavigationtype           := <choose><when test="navigationMode != null">#{navigationMode,jdbcType=VARCHAR}</when><otherwise>'UT'</otherwise></choose>     
```

Comment:
Defaulting to UT? I think that's a reasonable default, but I'm not sure that we should default... So in the case the behavior for a base/navigate/comid/ without any additional information would be to return everything upstream?